src/main/scala/io/digitallibrary/bookapi/repository/CategoryRepository.scala

```diff
@@ -24,6 +24,13 @@ trait CategoryRepository {
         .apply.headOption
     }
 
+    def withId(id: Long)(implicit session: DBSession = AutoSession): Option[Category] = {
+      sql"select ${cat.result.*} from ${Category.as(cat)} where ${cat.id} = ${id}"
```

Comment:
En spesiell grunn til at du har brukt sql"select ..." og ikkje select.from? Eg liker den siste best fordi du fjerner muligheten for skrivefeil i sql. Sjølv om det er er liten mulighet for skrivefeil i ei linje sql.

Reply:
Fikset.